manifests-kitchen/bitbar.pp

```diff
@@ -0,0 +1,29 @@
+##################################################
+#### MOCK CLASSES WHICH SHOULD NOT TESTED HERE
+# class kitchen_template2(
+#   Hash $config = {},
+# ) {
+#   notice( 'mocked class ==> kitchen_template::foobar' )
+# }
+
+# dirty hack: include site.pp
+# can't point at directory of manifests, as last check in site.pp fails for test-kitchen
```

Comment:
is this an issue with kitchen, or with our puppet? (can we create an issue to fix the cause or is it unlikely to affect other platforms?)

Reply:
I don't think puppet should have deprecated `import` (the recommend alternative is to use include (but that only works for modules) or to use a directory of manifests). In normal puppet use cases, you don't need it, but wIth puppet-kitchen it would be very useful to compose manifests.

I've figured out a workaround using symlinks. :)
I can't think of a fix we could ask for upstream (other than re-enabling import). We could refactor our code to not use these globals... but I guess it's the only way to set 

I think we should settings like this should live in profiles (or hiera if we really want to set something globally).